app/models/person.rb

```diff
@@ -15,7 +15,8 @@ class Person < ActiveRecord::Base
 
   before_validation :initialize_person
   validates_associated :mail_address
-  validates :mail_address, presence: true
+  validates :mail_address, :dob, presence: true
```

Comment:
The `presence: true` means that a person can't be saved to the database unless a birthday has been entered.  That's not right.  We want to allow saving people without birthdays.  Keep in mind that users should be able to leave any part of the form blank and come back to their work later.